aspnet/signalr/overview/getting-started/tutorial-high-frequency-realtime-with-signalr.md

```diff
@@ -1,222 +1,276 @@
-
-While the application will function using this method, this is not a recommended programming model, since there would be no upper limit to the number of messages getting sent, so the clients and server could get overwhelmed with messages and performance would degrade. The displayed animation on the client would also be disjointed, as the shape would be moved instantly by each method, rather than moving smoothly to each new location. Later sections of the tutorial will demonstrate how to create a timer function that restricts the maximum rate at which messages are sent by either the client or server, and how to move the shape smoothly between locations. The final version of the application created in this tutorial can be downloaded from [Code Gallery](https://code.msdn.microsoft.com/SignalR-20-MoveShape-Demo-6285b83a).
-
-This tutorial contains the following sections:
-
-- [Prerequisites](#prerequisites)
-- [Create the project and add the SignalR and JQuery.UI NuGet package](#createtheproject2013)
-- [Create the base application](#baseapp)
-- [Starting the hub when the application starts](#startup2013)
-- [Add the client loop](#clientloop)
-- [Add the server loop](#serverloop)
-- [Add smooth animation on the client](#animation)
-- [Further Steps](#furthersteps)
-
-<a id="prerequisites"></a>
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Set up the project
+> * Create the base application
+> * Map to the hub when app starts
+> * Add the client
+> * Run the app
+> * Add the client loop
+> * Add the server loop
+> * Add smooth animation
+
+[!INCLUDE [Consider ASP.NET Core SignalR](~/includes/signalr/signalr-version-disambiguation.md)]
 
 ## Prerequisites
 
-This tutorial requires Visual Studio 2013.
+### Visual Studio 2017
+
+If you don't already have it, [download Visual Studio 2017](https://visualstudio.microsoft.com/downloads/) now.
+
+### ASP.NET and web development workload
+
+When Visual Studio Installer opens, make sure you install the **ASP.NET and web development** workload. It's under **Web & Cloud**.
+
+### NuGet package source API
+
+You'll need to set up the NuGet API to install the jQuery UI package.
+
+1. When Visual Studio 2017 opens, select **Tools** > **Options**.
+
+1. In **Options**, select **NuGet Package Manager** > **Package Sources**.
+
+1. Next to **Available package sources**, select **+** and enter this information:
 
-<a id="createtheproject2013"></a>
+    | Setting | Value |
+    | ------- | ----- |
+    | Name | Enter *nuget.org* |
+    | Source | Enter *https://api.nuget.org/v3/index.json* |
 
-## Create the project and add the SignalR and JQuery.UI NuGet package
+1. Select **OK**.
 
-In this section, we'll create the project in Visual Studio 2013.
+## Set up the project
 
-The following steps use Visual Studio 2013 to create an ASP.NET Empty Web Application and add the SignalR and jQuery.UI libraries:
+In this section, we'll create the project in Visual Studio 2017.
```

Comment:
Avoid first person

Reply:
And future tense. It should be

In this section, you create the project in Visual Studio 2017.